randovania/games/samus_returns/logic_database/Area 1.txt

```diff
@@ -286,9 +293,35 @@ Extra - asset_id: collision_camera_014
       All of the following:
           Morph Ball
           Any of the following:
-              Ice Beam or Space Jump or Simple IBJ
-              Spider Ball and Lay Any Bomb
-              High Jump Boots and Gravity Suit
+              Space Jump
+              All of the following:
+                  # Spider Ball up
+                  Spider Ball
+                  Any of the following:
+                      Lay Any Bomb
+                      All of the following:
+                          # Shoot Block before spidering up
+                          Precise Aiming (Beginner)
+                          Mid-Air Morph (Beginner) or Use Spring Ball
```

Comment:
Mid-Air Morph should be grouped with `Precise Aiming`, and then the options under that should be `Phase Drift or Movement (Intermediate)`.

We also decided Spring Ball isn't necessary since you can just Spider around the top of the room using the Bomb block.

Reply:
My idea was even with the midair Morph you either need to move faster (movement) or use phase Drift to get to the item in time. That was tested last night. If there's enough time to reach the item withbphase Drift and slider and without a mid-air Morph, then spring isn't necessary. Would need to test that last bit.

Reply:
Ah, I thought we didn't want to include movement for the spidering up part. Gotcha.